packages/ui/src/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,30 @@
+import {
+  Tooltip as AriaTooltip,
+  TooltipProps as ReactAriaTooltipProps,
+  TooltipTrigger
+} from "react-aria-components";
+import Image from "@repo/ui/image";
+
+import "./Tooltip.css";
+
+export interface TooltipProps extends ReactAriaTooltipProps {
+  tooltipContent: React.ReactNode;
```

Comment:
I'm getting a warning that React isn't directly imported here. I know it won't actually throw any errors but I do like to import { ReactNode} from 'react' for this and just make this ReactNode. YMMV.